data/utils.py

```diff
@@ -3,8 +3,6 @@
 import torchvision.transforms as transforms
 import torch
 
-from continuum.datasets import PyTorchDataset
-from continuum import TaskSet
```

Comment:
Just checking these are not deleted by accident

Reply:
All good, he moved these to train.py